LightWeightSignalBus.cs

```diff
@@ -15,8 +15,6 @@ public static IDisposable Subscribe(Action<T> action)
 
 	public static void Fire(T t) => action?.Invoke(t);
 
-	public static void AbstractFire(T t) => SignalBus.AbstractFire<T>(t);
```

Comment:
This is nice, but as you said, this script may be needed to be to stay in a single repo, since it would make it not being lightweight, and the lightweight would be required to be coupled with the interface version, i think this script may need to be in its own folder, with its own asmdef, to avoid, coupling with the interface compatible one,

it wouldnt make sense using this one when using the interface compatible one, since its like this one but with more features, this one is just for easy usage and understanding on how the signalbus system works from the very base, i would just undo this addition, to keep the usages of both systems separated